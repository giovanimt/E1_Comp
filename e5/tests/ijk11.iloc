loadI 400 => r0 
addI rfp, 4 => r1 
store r0 => r1 
loadI 2 => r2 
addI rfp, 8 => r3 
store r2 => r3 
addI rfp, 8 => r14 
load r14 => r15 
addI rfp, 4 => r12 
load r12 => r13 
div r13, r15 => r16 
addI rfp, 8 => r8 
load r8 => r9 
addI rfp, 4 => r6 
load r6 => r7 
div r7, r9 => r10 
addI rfp, 4 => r4 
load r4 => r5 
add r5, r10 => r11 
add r11, r16 => r17 
addI rfp, 0 => r18 
store r17 => r18 
