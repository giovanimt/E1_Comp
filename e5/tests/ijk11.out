00018697 800
00018701 400
00018705 2
